### ISSUE TITLE
ImportError: No module named builtins

### ISSUE THREAD

**Issue by friederbluemle** [facebook/PathPicker#305]
I just installed `fpp` using `brew install fpp`. When I try to run it, this is the output:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/fpp/0.8.2/libexec/src/processInput.py", line 12, in <module>
    import format
  File "/usr/local/Cellar/fpp/0.8.2/libexec/src/format.py", line 13, in <module>
    from formattedText import FormattedText
  File "/usr/local/Cellar/fpp/0.8.2/libexec/src/formattedText.py", line 9, in <module>
    from colorPrinter import ColorPrinter
  File "/usr/local/Cellar/fpp/0.8.2/libexec/src/colorPrinter.py", line 7, in <module>
    import output
  File "/usr/local/Cellar/fpp/0.8.2/libexec/src/output.py", line 8, in <module>
    from builtins import str
ImportError: No module named builtins
```

Am I missing anything?

**Comment by pcottle** [comment]
Seems like an issue with `libexec`... what version of python are you using? can you try python 3?

**Comment by friederbluemle** [comment]
(before installing fpp)

$ python --version
Python 2.7.10

Now:

$ python --version
Python 2.7.16

It seems like installing fpp installed a newer version of python at the same time. How do I switch to Python 3 (temporarily)? I read that it might not be the best idea to set the default python on macOS to Python 3.

**Comment by pcottle** [comment]
﻿Just include --python3 and it’ll use that command instead!

That’s odd that it changed your version, usually homebrew doesn’t do that…

**Comment by friederbluemle** [comment]
That works! Thanks @pcottle! :+1:

What I ended up doing is edit `/usr/local/Cellar/fpp/0.8.2/libexec/fpp`, and replace

```
PYTHONCMD="python"
```

with

```
PYTHONCMD="python3"
```

**Comment by pcottle** [comment]
Sweet! Yeah sorry about this oddity but at least you're unblocked for now.

**Comment by laoshaw** [comment]
I think fpp really should default to python3 instead of using python nowadays

**Comment by pcottle** [comment]
Definitely a good suggestion, we would just need to change the homebrew recipe. I’m not sure if python3 or python2 is still the more popular option there…

**Comment by markstos** [comment]
I got this same failure when installing the latest version on Debian 8 where "python" also resolves to python 2.7. If I use the --python3 option, it works. I second the recommendation to switch to Python 3 as the default, especially since Python 2.7 supports appears broken out of the box in both Mac and Linux environments.

**Comment by pcottle** [comment]
merged in!

**Comment by pcottle** [comment]
new release:
https://github.com/facebook/PathPicker/releases/tag/0.9.2

**Comment by pcottle** [comment]
new PR for homebrew:
https://github.com/Homebrew/homebrew-core/pull/43673

**Comment by friederbluemle** [comment]
Awesome, thanks @pcottle 👍
